[PG-644] Add pgsm_track_application_names parameter description (#63)
percona/pgsm-docs: docs/setup.md

    >
    >`pg_stat_monitor` **must** follow `pg_stat_statements`. For example, `ALTER SYSTEM SET shared_preload_libraries = 'foo, pg_stat_statements, pg_stat_monitor'`.

    !!! warning
   

>        It's make sence to disable application name tracking as it's expensive feature and may cause perfomance degradation proportional to connections number.

Reviewer: ```suggestion
       It makes sense to disable the application name tracking because it may cause perfomance degradation proportional to the number of connections.
```

       ```sql
       ALTER SYSTEM SET pg_stat_monitor.pgsm_track_application_names = 'no';

>        ```

Author: @nastena1606 I need your help here, how to make this text part of the warning frame?

    >
    >`pg_stat_monitor` **must** follow `pg_stat_statements`. For example, `ALTER SYSTEM SET shared_preload_libraries = 'foo, pg_stat_statements, pg_stat_monitor'`.

    !!! warning
   

>         It's make sence to disable application name tracking as it's expensive feature and may cause perfomance degradation proportional to connections number.

Author: ```suggestion
        It makes sense to disable the application name tracking because it may cause perfomance degradation proportional to the number of connections.
```
